### PR TITLE
Handle login failures #37

### PR DESCRIPTION
**Pull request by rathore-abhay**
Handling error response from strapi and displaying it as alert for now.

### CODE REVIEW

File: client/pages/admin/signin.js [hunk 1]
```diff
@@ -27,15 +27,24 @@ const AdminSignin = () => {
...
-    const userCookie = Cookies.get('username');
-    if (userCookie) {
-      setUser(userCookie);
-      router.push('/admin/meetups');
-    }
+    })
+      .then((response) => {
+        return response.json()
+      })
+      .then((jsonResponse) => {
```
Comment (samanthakem) [inline client/pages/admin/signin.js:34]: Could you use `catch` instead?

https://medium.com/@lucymarmitchell/using-then-catch-finally-to-handle-errors-in-javascript-promises-6de92bce3afc

File: client/pages/admin/signin.js [hunk 2]
```diff
@@ -27,15 +27,24 @@ const AdminSignin = () => {
...
+        if (jsonResponse.error)
+          throw Error(jsonResponse.data[0].messages[0].message);
+        else {
```
Comment (samanthakem) [inline client/pages/admin/signin.js:37]: Why are you re-setting the token if the login is wrong?

**Review by samanthakem — CHANGES_REQUESTED** [review]
Hey there! Thank you for your contribution!

I added a few comments, please ping me if you have questions on why I added those.

**Comment by nathanhannig** [comment]
I feel this can now be closed as #53 implements the same fix

**Pull request closed without merging**